src/components/menu/styles.ts

```diff
@@ -38,6 +38,9 @@ const baseMenuItemStyles = `
     color: var(--text-white);
     background: var(--fill-action-dark);
   }
+  &:focus-visible {
+    outline: none;
+  }
```

Comment:
the button have default outline color when focused (using `tab` key for example), but anchor doesn't have, so this is just to make the style the same (use defined `:focus` style)